analytic_production_cost_report/report/analytic_production_cost_report.py

```diff
@@ -0,0 +1,74 @@
+    name = fields.Char(string='Description', readonly=True)
+    partner_id = fields.Many2one('res.partner', string='Partner')
+    company_id = fields.Many2one(
+        'res.company', string='Company', required=True)
+    currency_id = fields.Many2one(
+        'res.currency', string='Currency', required=True)
+    account_id = fields.Many2one(
+        'account.analytic.account', string='Account', required=True)
+    general_account_id = fields.Many2one(
+        'account.account', string='General Account', required=True)
+    journal_id = fields.Many2one(
+        'account.analytic.journal', string='Journal', required=True)
+    move_id = fields.Many2one(
+        'account.move.line', string='Move', required=True)
+    product_id = fields.Many2one(
+        'product.product', string='Product', required=True)
+    product_uom_id = fields.Many2one(
+        'product.uom', string='Product Unit of Measure', required=True)
+    production = fields.Many2one(
+        'mrp.production', string='Production', readonly=True)
+    workorder = fields.Many2one(
+        'mrp.production.workcenter.line', string='Work order', readonly=True)
+    amount = fields.Float(string='amount', readonly=True)
+    unit_amount = fields.Integer(string='Unit Amount', readonly=True)
+    nbr = fields.Integer(string='# Entries', readonly=True)
+
+    def init(self, cr):
+        tools.drop_view_if_exists(cr, 'analytic_production_cost_report')
+        cr.execute("""
+            create or replace view analytic_production_cost_report as (
+                 select
+                     min(a.id) as id,
+                     count(distinct a.id) as nbr,
+                     a.date as date,
+                     a.user_id as user_id,
+                     a.name as name,
+                     analytic.partner_id as partner_id,
+                     a.company_id as company_id,
+                     a.currency_id as currency_id,
+                     a.account_id as account_id,
+                     a.general_account_id as general_account_id,
+                     a.journal_id as journal_id,
+                     a.move_id as move_id,
+                     a.product_id as product_id,
+                     a.product_uom_id as product_uom_id,
+                     a.mrp_production_id as production,
+                     a.workorder as workorder,
+                     sum(a.amount) as amount,
+                     sum(a.unit_amount) as unit_amount
+                 from
+                     account_analytic_line a, account_analytic_account analytic
+                 where analytic.id = a.account_id
+                 group by
+                     a.date, a.user_id, a.name, analytic.partner_id,
+                     a.company_id, a.currency_id, a.account_id,
+                     a.general_account_id, a.journal_id, a.move_id,
+                     a.product_id, a.product_uom_id, a.mrp_production_id,
+                     a.workorder
```

Comment:
@pedrobaeza, @anajuaristi me dice que NO hay que limitar los apuntes analíticos que tengan mrp_production_id a no nulo, esto es porque hay apuntes que pertenecen al proyecto de una OF, y los cualesn o tienen informado este campo.
¿Porque me dices que estoy asumiento que tengo instalado mrp_operations_extension?, la única dependencia que tiene este módulo es con "mrp_project_link", ya que es este módulo el que crea los campos de "mrp_production_id, "workorder" en el modelo account.analytic.line